pep-0682.rst

```diff
@@ -159,6 +165,12 @@ the new option, %-formatting cannot.  There is already precedent for not
 extending %-formatting with new options, as was the case for the
 ``,`` option (:pep:`378`).
 
+C99 ``printf`` is already employing the ``z`` option character for another
```

Comment:
simpler to read. I'd also avoid the colon in the next line (169) if possible, but it is easier to parse.
```suggestion
C99 ``printf`` already uses the ``z`` option character for another
```